vm/interpreter/src/gas_block_store.rs

```diff
@@ -8,7 +8,7 @@ use fvm_ipld_blockstore::Blockstore;
 use std::cell::RefCell;
 use std::rc::Rc;
 
-/// Blockstore wrapper to charge gas on reads and writes
+/// `GasBlockStore` wrapper to charge gas on reads and writes
```

Comment:
`GasBlockStore` wraps `BlockStore`.

```suggestion
/// `BlockStore` wrapper to charge gas on reads and writes
```